### PR TITLE
Upstream adjustments

### PR DESCRIPTION
**Pull request by ericherman**
This set of changes adjusts files to point to the new repository location,
without attempting to address branding and topics raised in #1024.

### CODE REVIEW

File: publiccode.yml [hunk 1]
```diff
@@ -43,32 +43,26 @@ description:
     videos:
       - 'https://www.youtube.com/watch?v=QWt6vB-cipE'
 developmentStatus: beta
-landingURL: 'https://standard.publiccode.net/'
+landingURL: 'https://standard-for-public-code.github.io/standard-for-public-code/'
 legal:
-  authorsFile: 'https://raw.githubusercontent.com/publiccodenet/standard/master/AUTHORS.md'
+  authorsFile: 'https://raw.githubusercontent.com/standard-for-public-code/standard-for-public-code/master/AUTHORS.md'
   license: CC0-1.0
   mainCopyrightOwner: Foundation for Public Code
```
Comment (jgroenen) [inline publiccode.yml:50]: I think I saw this replaced with "the contributors" or something elsewhere?

File: publiccode.yml [hunk 2]
```diff
@@ -43,32 +43,26 @@ description:
...
-  repoOwner: Foundation for Public Code
+  repoOwner:
```
Comment (jgroenen) [inline publiccode.yml:51]: Should this be the Maintenance Board or something?

**Review by jgroenen — COMMENTED** [review]
I only checked that there were no strange changes, not for technical functionality or anything...

File: publiccode.yml (hunk 1 above)
Comment (jgroenen) [reply publiccode.yml:50]: This comment is about line 50 "mainCopyrightOwner"

Comment (clausmullie) [inline publiccode.yml:50]:
```suggestion
  mainCopyrightOwner: The contributors
```

Comment (clausmullie) [reply publiccode.yml:50]:
```suggestion
  mainCopyrightOwner: The contributors
```

**Comment by clausmullie** [comment]
Scanned through all the changes as well - all the link changes are correct, and the automated link check passes, so pending the two changes to copyright owner and repo owner, I think this is good to go.

**Review by zorp — APPROVED** [review]
Scanned through changes and everything looks correct.

**Pull request merged**